go.mod

```diff
@@ -101,6 +101,7 @@ require (
 	github.com/sergi/go-diff v1.2.0
 	github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749 // indirect
 	github.com/shurcooL/vfsgen v0.0.0-20200824052919-0d455de96546
+	github.com/snabb/sitemap v1.0.0 // indirect
```

Comment:
I think we should not import such a small library(that hasn't any test), I think we rather should make our own in modules/sitemap. This depedencies imports another dependencies which we won't use as well.